docker-compose.yaml

```diff
@@ -4,7 +4,7 @@ services:
     container_name: idw-keria
     restart: unless-stopped
     build:
-      context: https://github.com/WebOfTrust/keria.git#c9d35f901e293f166d5abbd58cda249248555253
+      context: https://github.com/WebOfTrust/keria.git#e8bc173437ac91a88abebc3bc8c6b7f07885da68
```

Comment:
please update everywhere that had `c9d35f901e293f166d5abbd58cda249248555253` - there's 3 places

Reply:
does this githash have the problem with Falcon? if so ill need to push a fix first

Reply:
Yes, this version had a problem with Falcon, I think we need to update it on Keria before merging this PR

Reply:
let me create a KERIA PR

Reply:
this needs to be the latest version now with falcon fix